### PR TITLE
Update document for manual installation

### LINKED ISSUE #46: Update manual install doc
The manual install doc should source the contents predominantly from https://github.com/RedHatOfficial/ocp4-helpernode/blob/master/docs/quickstart-ppc64le.md

### PR DESCRIPTION
**Pull request by bpradipt**
Fixes: #46
Signed-off-by: bpradipt@in.ibm.com

### CODE REVIEW

File: docs/ocp4-manual-deployment.md [hunk 1]
```diff
@@ -1,567 +1,209 @@
...
 
 ```
-while [ $(oc get configs.imageregistry.operator.openshift.io/cluster | wc -l) == 0 ]; do sleep 30; done
-oc patch configs.imageregistry.operator.openshift.io cluster --type merge --patch '{"spec":{"storage":{"emptyDir":{}}, "managementState": "Managed"}}'
+openshift-install wait-for bootstrap-complete --log-level debug
 ```
-**9.  Complete the installation**
 
-Once all the cluster Operators are AVAILABLE, run
+## **Finish Install
```
Comment (yussufsh) [inline docs/ocp4-manual-deployment.md]: missing **

File: docs/ocp4-manual-deployment.md [hunk 2]
```diff
@@ -1,567 +1,209 @@
...
+Watch your CSRs. These can take some time; go get come coffee or grab some lunch. You'll see your nodes' CSRs in "Pending" (unless they were "auto approved", if so, you can jump to the wait-for install-complete step)
+```
+watch oc get csr
+```
+To approve them all in one shot...
+```
+oc get csr --no-headers | awk '{print $1}' | xargs oc adm certificate approve
 ```
-INFO Waiting up to 30m0s for the cluster to initialize...
+Check for the approval status (it should say "Approved,Issued")
```
Comment (yussufsh) [inline docs/ocp4-manual-deployment.md]: CSRs are created not in any order or all at once. We should mention that approve all pending CSR until all the worker nodes are in "Ready" status.

Comment (bpradipt) [reply docs/ocp4-manual-deployment.md]: true.. thanks for pointing it out

File: docs/ocp4-manual-deployment.md (hunk 1 above)
Comment (bpradipt) [reply docs/ocp4-manual-deployment.md]: done

**Review by sudeeshjohn — COMMENTED** [review]
Since this doc is on DHCP way of installation, I think its good to  change the file name or a title mentions that

**Review by sudeeshjohn — APPROVED** [review]
LGTM

**Review by yussufsh — APPROVED** [review]

**Pull request merged**